edge-apps/countdown-timer/index.html

```diff
@@ -1,88 +1,62 @@
-          <div class="container-minute">
-            <h3 class="minute">0</h3>
-            <h3>Minute</h3>
-          </div>
-          <div class="container-second">
-            <h3 class="second">0</h3>
-            <h3>Second</h3>
-          </div>
-        </div>
-      </div>
-      <div class="date-time">
-        <div class="date">
-          <img src="static/images/icons/calendar.svg" alt="calendar icon" />
-          <span id="date"></span>
-        </div>
-        <div class="time">
-          <img src="static/images/icons/clock.svg" alt="clock icon" />
-          <span id="time"></span>
-        </div>
-      </div>
-    </section>
-  </div>
-
-
-
-
-
-  <script src="static/js/moment-with-locales.min.js"></script>
-  <script src="static/js/tz.min.js"></script>
-  <script src="static/js/moment-timezone-with-data.min.js"></script>
-  <script src="static/js/country-locale-map.1.18.5.min.js"></script>
-  <script src="static/js/offline-geocode-city-1.0.2.min.js"></script>
-  <script src="static/js/main.js"></script>
+	<script src="screenly.js?version=1"></script>
+
+	<div class="main-container">
+		<div class="primary-card">
+			<span class="primary-card-message message">Countdown</span>
+			<span class="primary-card-number day">0</span>
+			<span class="primary-card-text">DAYS</span>
+		</div>
+		<div class="secondary-container">
+			<div class="row-container">
+				<div class="secondary-card">
+					<object id="svgObject1"  class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number hour">0</span>
+					<span class="secondary-card-label">HOURS</span>
+				</div>
+				<div class="secondary-card">
+					<object id="svgObject1"  class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number minute">0</span>
+					<span class="secondary-card-label">MINUTES</span>
+				</div>
+					<div class="secondary-card">
+					<object id="svgObject1" class="clock-icon" data="./static/img/clock-icon.svg" type="image/svg+xml"></object>
+					<span class="secondary-card-number second">0</span>
+					<span class="secondary-card-label">SECONDS</span>
+				</div>
+			</div>
+			<div class="row-container">
+				<div class="secondary-card info-card">
+					<img id="brandLogo" src="" class="brand-logo" alt="Brand Logo" />
+					<span class="info-text">Powered by Screenly</span>
+				</div>
+				<div class="secondary-card">
+					<div class="secondary-card date-card">
+						<span class="date-text">WED</span>
+						<span class="date-number">0</span>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<script src="static/js/country-locale-map.1.18.5.min.js"></script>
+	<script src="static/js/moment-with-locales.min.js"></script>
+	<script src="static/js/moment-timezone-with-data.min.js"></script>
+	<script src="static/js/offline-geocode-city-1.0.2.min.js"></script>
+	<script src="static/js/tz.min.js"></script>
```

Comment:
Is tz.min.js library different from moment-timezone? Do we need to use both?

Reply:
Hi @iamplugged, The **tz.min.js** fetches the timezone as per the latitude latitude when override_timezone values are invalid. 

And in **moment-timezone**, it manages the locale time and date information for "Language-specific formatting".

Reply:
@iamplugged is it ok? Thanks.

Reply:
@salmanfarisvp Yes I've approved it.